src/sfctl/custom_container.py

```diff
@@ -0,0 +1,81 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric container support"""
+
+from __future__ import print_function
+import json
+import jsonpickle
+from azure.servicefabric.models import ContainerApiRequestBody
+
+def invoke_api( # pylint: disable=too-many-arguments
+        client,
+        node_name,
+        application_id,
+        service_manifest_name,
+        code_package_name,
+        code_package_instance_id,
+        container_api_uri_path,
+        container_api_http_verb=None,
+        container_api_content_type=None,
+        container_api_body=None,
+        timeout=60):
+    """Invoke container API on a cluster node"""
+
+    request_body = ContainerApiRequestBody(
+        container_api_uri_path,
+        container_api_http_verb,
+        container_api_content_type,
+        container_api_body)
+
+    response = client.invoke_container_api(
+        node_name,
+        application_id,
+        service_manifest_name,
+        code_package_name,
+        code_package_instance_id,
+        request_body,
+        timeout)
+
+    print(format_response(response))
```

Comment:
Why print instead of returning?

Reply:
With return, the output has special characters like quotes, \n ... , while the format with print is cleaner, as in the "examples " above.

> sfctl container logs --node-name N0020 --application-id nodejs1 --service-manifest-name NodeOnSF --code-package-name Code --code-package-instance-id 131673596679688285
"Running on http://localhost:8085\n"

(ENV) [15:44:38.24] leikong@SYLVIA X:\git\service-fabric-cli
> sfctl container invoke-api --node-name N0020 --application-id nodejs1 --service-manifest-name NodeOnSF --code-package-name Code --code-package-instance-id 131673596679688285 --container-api-uri-path "/containers/{id}/update" --container-api-http-verb=POST --container-api-body "{ \"CpuShares\": 512}"
"{\n    \"container_api_result\": {\n        \"body\": \"{\\\"Warnings\\\":null}\\n\",\n        \"content_encoding\": null,\n        \"content_type\": \"application/json\",\n        \"status\": 200\n    }\n}"